instagram/src/components/Login/Login.js

```diff
@@ -11,7 +12,7 @@ class Login extends Component {
 
 
     handleInputChange = e => {
-        this.setState({[e.target.name]: e.target});
+        this.setState({[e.target.name]: e.target.value});
```

Comment:
Good line to remember for the future. Ryan also went over this portion for forms via the search functionality in today's lecture. 8/2/2018.